### PR TITLE
Bug remediated

### PR DESCRIPTION
**Pull request by CRMTravis**

### CODE REVIEW

File: force-app/main/default/aura/simpleloancalculator/simpleloancalculatorController.js [hunk 1]
```diff
@@ -134,7 +134,7 @@
           Existing_Monthly_Credit_Payment__c: component
             .find("existingmonthlycreditpayment")
             .get("v.value"),
-          Total_Monthly__c: component
```
Comment (RemarioCRM) [inline force-app/main/default/aura/simpleloancalculator/simpleloancalculatorController.js:137]: why remove Total_Monthly__c?

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

File: force-app/main/default/aura/simpleloancalculator/simpleloancalculatorController.js (hunk 1 above)
Comment (CRMTravis) [reply force-app/main/default/aura/simpleloancalculator/simpleloancalculatorController.js:137]: I changed it to Proposed_Starting_Limit__c

Comment (CRMTravis) [reply force-app/main/default/aura/simpleloancalculator/simpleloancalculatorController.js:137]: It was saving to the wrong field

Comment (RemarioCRM) [reply force-app/main/default/aura/simpleloancalculator/simpleloancalculatorController.js:137]: add a null check

Comment (CRMTravis) [reply force-app/main/default/aura/simpleloancalculator/simpleloancalculatorController.js:137]: where exactly?

**Pull request merged**